proposals/lip-0038.md

```diff
@@ -295,48 +301,48 @@ Invalid cases:
 # public key is identity point in the group G1
 pk = 0xc00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
 proof = 0xc00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
-PopVerify(pk, proof) = INVALID
+PopVerify(pk, proof) == INVALID
 
 # pk does not represent a point on the curve E1
 pk = 0xa53d21a4cfd562c469cc81514d4ce5a6b577d8403d32a394dc265dd190b47fa9f829fdd7963afdf972e5e77854051f6f
 proof = 0x88bb31b27eae23038e14f9d9d1b628a39f5881b5278c3c6f0249f81ba0deb1f68aa5f8847854d6554051aa810fdf1cdb02df4af7a5647b1aa4afb60ec6d446ee17af24a8a50876ffdaf9bf475038ec5f8ebeda1c1c6a3220293e23b13a9a5d26
-PopVerify(pk, proof) = INVALID
+PopVerify(pk, proof) == INVALID
 
 # pk represents a point on the curve E1 but is NOT an element in the group G1
 pk = 0x960003aaf1632b13396dbad518effa00fff532f604de1a7fc2082ff4cb0afa2d63b2c32da1bef2bf6c5ca62dc6b72f9c
 proof = 0x88bb31b27eae23038e14f9d9d1b628a39f5881b5278c3c6f0249f81ba0deb1f68aa5f8847854d6554051aa810fdf1cdb02df4af7a5647b1aa4afb60ec6d446ee17af24a8a50876ffdaf9bf475038ec5f8ebeda1c1c6a3220293e23b13a9a5d26
-PopVerify(pk, proof) = INVALID
+PopVerify(pk, proof) == INVALID
 
 # proof is not a point on the curve E2 (tampered proof)
 pk = 0xa491d1b0ecd9bb917989f0e74f0dea0422eac4a873e5e2644f368dffb9a6e20fd6e10c1b77654d067c0618f6e5a7f79a
 proof = 0xb803eb0ed93ea10224a73b6b9c725796be9f5fefd215ef7a5b97234cc956cf6870db6127b7e4d824ec62276078e787db05584ce1adbf076bc0808ca0f15b73d59060254b25393d95dfc7abe3cda566842aaedf50bbb062aae1bbb6ef3b1fffff
-PopVerify(pk, proof) = INVALID
+PopVerify(pk, proof) == INVALID
 
 # proof is a point on the curve E2 but not the matching proof for pk
 pk = 0xa491d1b0ecd9bb917989f0e74f0dea0422eac4a873e5e2644f368dffb9a6e20fd6e10c1b77654d067c0618f6e5a7f79a
 proof = 0x88bb31b27eae23038e14f9d9d1b628a39f5881b5278c3c6f0249f81ba0deb1f68aa5f8847854d6554051aa810fdf1cdb02df4af7a5647b1aa4afb60ec6d446ee17af24a8a50876ffdaf9bf475038ec5f8ebeda1c1c6a3220293e23b13a9a5d26
-PopVerify(pk, proof) = INVALID
+PopVerify(pk, proof) == INVALID
 ```
 
 #### `signBLS`
 
 ```python
 sk = 0x263dbd792f5b1be47ed85f8938c0f29586af0d3ac7b977f21c278fe1462040e3
 tag = 0x4c534b5f54585f # ASCII-encoding of "LSK_TX_"
-networkIdentifier = 0x9ee11e9df416b18bf69dbd1a920442e08c6ca319e69926bc843a561782ca17ee
+chainID = 0x9ee11e9df416b18bf69dbd1a920442e08c6ca319e69926bc843a561782ca17ee
```

Comment:
Here the `chainID` is not 4-bytes, but I guess it should not impact the test.

Reply:
Indeed, I prefer not to change these test vectors that were prepared by Andreas